### PR TITLE
Add zachjs-sv2v package

### PR DESCRIPTION
**Pull request by piotr-binkowski**
Aim of this PR is to create conda package with [sv2v](https://github.com/zachjs/sv2v) tool which will be used in SymbiFlow/sv-tests#183

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -38,6 +38,8 @@ env:
   - PACKAGE=icestorm
   - PACKAGE=iverilog
   - PACKAGE=verilator
+  # systemverilog tools
```
Comment (mithro) [inline .travis.yml]:
```suggestion
  # SystemVerilog tools
```

**Comment by mithro** [comment]
Does this look right?
```
bin/zachjs-sv2v
info/about.json
info/files
info/git
info/hash_input.json
info/index.json
info/licenses/LICENSE
info/paths.json
info/recipe/build.sh
info/recipe/conda_build_config.yaml
info/recipe/meta.yaml
info/recipe/meta.yaml.template
info/recipe/recipe_append.yaml
info/test/run_test.sh
```

File: .travis.yml (hunk 1 above)
Comment (piotr-binkowski) [reply .travis.yml]: done

**Comment by piotr-binkowski** [comment]
Package contents look OK

**Review by mithro — APPROVED** [review]

**Pull request merged**